app/views/trips/index.html.haml

```diff
@@ -1,3 +1,60 @@
+- content_for( :title ) do
+  - @title = t( :trips )
+  = strip_tags( @title )
 .container
-  - for trip in @trips
-    %p= link_to trip.title, trip
\ No newline at end of file
+  .row
+    .col-xs-12
+      %h2= @title
+      .panel.panel-info
+        .panel-heading
+          %h3.panel-title
+            = "About Trips"
+        .panel-body
+          %p
+            = "Trips describe survey effort by recording where and when an area was surveyed and what was targeted."
```

Comment:
FWIW, you could just do
```haml
%p
  Trips describe survey effort by recording where and when an area was surveyed
  and what was targeted. The existence of observations in the context of trips
  can be used to tabulate presence absence data. Presence absence data can be
  useful for understanding samplibg bias in species distribution modeling.
  = link_to "Read more", "/pages/trips"

Or you could globalize it if you think this needs to be translated.

Reply:
thanks - also not done internationalizing but will do that after 4/4